### PR TITLE
FUSETOOLS-2053 - fix problem with camel version preselection in wizard

### PR DESCRIPTION
**Pull request by lhein**

### CODE REVIEW

**Comment by bfitzpat** [comment]
+1

**Comment by lhein** [comment]
Its a fairly small change. The only thing I do here is to filter camel versions which don't fit the runtimes camel version in terms of branded vs community camel versions. It means if the runtime contains a Red Hat branded Camel version (it always will) then we only suggest Red Hat branded versions for that runtime and NOT like it is now 2.17.3 community version.

**Comment by apupier** [comment]
ok, I still think that a unit test will help to understand that

File: editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationWizardRuntimeAndCamelPageIT.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+/**
+ * @author lhein
+ *
+ */
+public class FuseIntegrationWizardRuntimeAndCamelPageIT {
+	
+	@Test
+	public void testCompatibleCamelVersionSelectedForUnsupportedBrandedVersion() throws Exception {
+		FuseIntegrationProjectWizard wiz = new FuseIntegrationProjectWizard();
```
Comment (apupier) [inline editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationWizardRuntimeAndCamelPageIT.java]: What about extracting these four lines in a setup @before method?

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
testPR

**Pull request merged**